apier/v1/apier.go

```diff
@@ -1086,8 +1085,8 @@ func (arrp *AttrRemoveRatingProfile) GetId() (result string) {
 }
 
 func (apiv1 *APIerSv1) RemoveRatingProfile(attr AttrRemoveRatingProfile, reply *string) error {
-	if (attr.Subject != utils.EmptyString && utils.IsSliceMember([]string{attr.Tenant, attr.Category}, "")) ||
-		(attr.Category != utils.EmptyString && attr.Tenant == utils.EmptyString) {
+	if attr.Subject != utils.EmptyString && utils.IsSliceMember([]string{attr.Tenant, attr.Category}, "") ||
```

Comment:
I don't think we should remove parentheses here since you get a different behavior than the one initially intended.